fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -262,6 +264,15 @@ public Property getProperty(final String relPath) {
     }
 
     @Override
+    public void setURIProperty(final String relPath, final URI value) {
```

Comment:
Can you add a Javadoc note like the one you have on this commit that indicates that no references are created with this method. That would be helpful in the future to let people understand the specialness of this method.